Client.xcodeproj/project.pbxproj

```diff
@@ -5569,13 +5573,16 @@
 					271DEC18234CC75F009DAC37 = {
 						CreatedOnToolsVersion = 11.1;
 						LastSwiftMigration = 1110;
+						ProvisioningStyle = Automatic;
```

Comment:
Im not sure I can control this one since its not part of the build settings. I believe Xcode added this once code sign style differed between different build configs